actor/v2action/service_instance_test.go

```diff
@@ -306,4 +306,251 @@ var _ = Describe("Service Instance Actions", func() {
 			})
 		})
 	})
+
+	Describe("GetSharedToSpaceGUID", func() {
+		var (
+			spaceGUID  string
+			warnings   Warnings
+			executeErr error
+
+			serviceInstanceName string
+			sourceSpaceGUID     string
+			sharedToOrgName     string
+			sharedToSpaceName   string
+		)
+
+		BeforeEach(func() {
+			sourceSpaceGUID = "some-source-space-guid"
+			serviceInstanceName = "some-service-instance"
+		})
+
+		JustBeforeEach(func() {
+			spaceGUID, warnings, executeErr = actor.GetSharedToSpaceGUID(serviceInstanceName, sourceSpaceGUID, sharedToOrgName, sharedToSpaceName)
+		})
+
+		Context("when the service instance exists", func() {
+			BeforeEach(func() {
+				fakeCloudControllerClient.GetSpaceServiceInstancesReturns(
+					[]ccv2.ServiceInstance{
+						{
+							GUID: "some-service-instance-guid",
+							Name: "some-service-instance",
+						},
+					},
+					ccv2.Warnings{"get-space-service-instances-warning"},
+					nil,
+				)
+			})
+
+			It("calls GetSpaceServiceInstance with the correct service instance name and space guid", func() {
+				spaceGUIDArg, _, queryArg := fakeCloudControllerClient.GetSpaceServiceInstancesArgsForCall(0)
```

Comment:
Please check call count before checking args, otherwise this could potentially throw a panic.